### DISCUSSION TITLE
[Suggestion] Include version number in the name of Mozc installer for Windows.

### DISCUSSION THREAD

**Discussion by mikunimaru** [google/mozc#886] [Ideas]
I appreciate the great update that allows creating an installer for Mozc on Windows. As there is no stable version of Mozc, there is a possibility of having to roll back the version of Mozc being used. If the version number (or commit ID) is included in the name of the output installer for Mozc, it would be easier to manage multiple versions of installers and confirm the reproducibility of bugs. Thank you for your consideration.

**Comment by hiroyuki-komatsu** [comment]
Hi mikunimaru,
I'm sorry for the late reply.

It makes sense to me. Please let us consider to find a good way to meet the requirements.
Thank you,
